services/predictors/vila/server/predictor.py

```diff
@@ -0,0 +1,46 @@
+import os
+from typing import List
+
+from pydantic import BaseSettings
+
+from mmda.predictors.hf_predictors.vila_predictor import SimpleVILAPredictor
+from mmda.types.document import Document
+from mmda.types.annotation import SpanGroup
+
+
+class PredictorConfig(BaseSettings):
+    """Uninitialized fields must be set via environment variables"""
+
+    ARTIFACTS_DIR: str
+    label_names = {2: "label2", 3: "label3"}
```

Comment:
So in the new VILA models, I've managed to store both the label_map and vila related configurations inside the model config file, e.g.,
- https://huggingface.co/allenai/ivila-block-layoutlm-finetuned-docbank/blob/main/config.json#L11
- https://huggingface.co/allenai/ivila-block-layoutlm-finetuned-docbank/blob/main/config.json#L55

And the code is to set automatically load the configurations, see:
- https://github.com/allenai/VILA/blob/e53cf1ef1edc8623162603b373e168f3d2aa1758/src/vila/predictors.py#L59
- https://github.com/allenai/VILA/blob/e53cf1ef1edc8623162603b373e168f3d2aa1758/src/vila/dataset/preprocessors/config.py#L11

These updates haven't been incorporated into the current mmda VILA predictors -- but should be easily implemented in the new few updates.